packages/framework/undo-redo/src/sequenceHandler.ts

```diff
@@ -124,6 +124,7 @@ export class SharedSegmentSequenceRevertible implements IRevertible {
                                     tracked.propertyDelta,
                                     undefined);
                             }
+                        // fallthrough
```

Comment:
This looks like it found a bug, like there should be a `break` statement above...?  @anthony-murphy, @wes-carlson what do you think?

Reply:
(If so, that change should come in a different PR)